tests/virtualization/universal/register_guests.pm

```diff
@@ -16,13 +16,20 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
-    $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    # $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    select_console 'root-console';
+    my %ltss_products = @{get_var_array("LTSS_PRODUCTS")};
 
     foreach my $guest (keys %virt_autotest::common::guests) {
         record_info "$guest", "Registrating $guest against SMT";
         my ($sles_running_version, $sles_running_sp) = get_os_release("ssh root\@$guest");
+        my $version_id = $sles_running_sp != 0 ? "$sles_running_version.$sles_running_sp" : $sles_running_version;
         if ($sles_running_version >= 12) {
             assert_script_run("ssh root\@$guest SUSEConnect -r " . get_var('SCC_REGCODE') . " -e " . get_var("SCC_EMAIL"));
+            sleep 10;
```

Comment:
Is sleep necessary? if it is required, a comment is welcome as it is not recommended in openqa coding rule.

Reply:
Sometime "SUSEConnect -r" takes a bit longer time to complete. I once saw it broke the next "SUSEConnect -r ltss" . I don't why that happened. I am not sure if it necessary for the moment,   just feel that it's safe to wait a few seconds to perform next register. 
I will add a comment there.